dataservice/api/common/schemas.py

```diff
@@ -46,10 +53,38 @@ def wrap_envelope(self, data):
                             'code': data['code']}}
 
 
+def response_generator(schema):
+    class RespSchema(Schema):
+        _status = fields.Dict(example={'message': 'success', 'code': 200})
+        results = fields.Nested(schema)
+
+    return RespSchema
+
+
+def paginated_generator(schema):
```

Comment:
So I'm not sure about this one, but currently the documentation renders `results` in the responses for get collection endpoints as a dict. I think it might be because you have `results = fields.Nested(schema)`. Does it need to be `results = fields.List(fields.Nested(schema)`? For example: `/participants` shows:

```python
{
"_links": {
"next": "?page=3",
"prev": "?page=1",
"self": "?page=2"
},
"_status": {
"code": 200,
"message": "success"
},
"limit": 10,
"results": {
"_links": "string",
"created_at": "2018-02-13T20:33:22Z",
"demographic": "string",
"diagnoses": [
"string"
],
"external_id": "string",
"kf_id": "string",
"modified_at": "2018-02-13T20:33:22Z",
"samples": [
"string"
]
},
"total": 1342
}
```